action.yml

```diff
@@ -48,15 +51,47 @@ inputs:
 runs:
   using: 'composite'
   steps:
-    - uses: actions/checkout@v2
-    - run: |
+    - name: Checkout the fresh project
+      uses: actions/checkout@v2
+
+    - name: Configure git
+      run: |
         git config --global user.email "${{ inputs.actor-email }}"
         git config --global user.name "${{ inputs.actor-name }}"
-        npm install
       shell: 'bash'
-    - uses: actions/github-script@v5
+
+    - name: Version bump
+      id: version-bump
+      if: ${{ github.event_name == 'workflow_dispatch' }}
+      run: |
+        npm version --no-git-tag-version ${{ inputs.semver }}
+        echo "::set-output name=NPM_VERSION::$(echo $(node -p "require('./package.json').version"))"
+      shell: 'bash'
+
+    - name: Build the package
+      if: ${{ inputs.build-command }}
+      run: |
+        node -v
+        npm -v
+        ${{ inputs.build-command }}
+      shell: 'bash'
+
+    - name: Commit and open PR
+      uses: actions/github-script@v5
+      if: ${{ github.event_name == 'workflow_dispatch' }}
+      with:
+        github-token: ${{ inputs.github-token }}
+        script: |
+          const script = require('${{ github.action_path }}/dist/index.js')
+          await script({ github, context, inputs: ${{ toJSON(inputs) }} })
+      env:
+        NPM_VERSION: ${{ steps.version-bump.outputs.NPM_VERSION }}
```

Comment:
ok no let's provide this as an input to the script please, not an env var which is inconsistent with the way we're providing other input